azure-pipelines.yml

```diff
@@ -152,6 +152,14 @@ jobs:
         publishLocation: Container
       condition: succeeded()
 
+    - task: PublishBuildArtifacts@1
+      displayName: Publish VSIX Packages
```

Comment:
Shouldn't this rather say `Publish installer`?

Reply:
I don't mind the current name, if for no other reason than the install script wrapping the VSIX package is only needed as a workaround for limitations (arguably bugs) in the VSIX installer/uninstaller. I also wouldn't object if someone sent a PR to rename this.